iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/RoutePreview/RoutePreviewStatus/BaseRoutePreviewStatus.swift

```diff
@@ -118,15 +118,16 @@ final class BaseRoutePreviewStatus: SolidTouchView {
     if MWMRouter.hasRouteAltitude() {
       heightBox.isHidden = false
       MWMRouter.routeAltitudeImage(for: heightProfileImage.frame.size,
-                                   completion: { image, elevation in
-                                     self.heightProfileImage.image = image
-                                     guard let elevation = elevation else { return }
-                                     let attributes: [NSAttributedString.Key: Any] =
-                                       [
-                                         .foregroundColor: UIColor.linkBlue(),
-                                         .font: UIFont.medium14()
-                                       ]
-                                     self.elevation = NSAttributedString(string: "▲▼ \(elevation)", attributes: attributes)
+                                    completion: { image, totalAscent, totalDescent in
+                                    self.heightProfileImage.image = image
+                                    if let totalAscent = totalAscent, let totalDescent = totalDescent {
+                                      let attributes: [NSAttributedString.Key: Any] =
```

Comment:
Can this constant be a part of the class and reused, to avoid unnecessary memory allocations on each moment?